### ISSUE TITLE
Telize API shutdown

### ISSUE THREAD

**Issue by JulianSamarjiev** [darul75/satelize#6]
Hey @darul75, first of all, thank you for this package.

It seems that the Telize API, which your package is using, is shut down and therefore Satelize is not functional, at least in my application.

My question is, should I look for an alternative package, or do you have plans on adapting it?

**Comment by darul75** [comment]
Hi, 
Thank you for feedback.
I made some changes long time ago...and maybe won't be able to maintain it but as Telize API was not free anymore, now an internal DB (extract from their free data) is loaded and should work for you too. https://github.com/darul75/satelize/blob/master/src/satelize.js#L25

**Comment by JulianSamarjiev** [comment]
@darul75 thank you for the quick reply, I'll try updating the package then, as I am currently running 0.1.2, anything I need to keep in mind? Thanks again!

**Comment by darul75** [comment]
I have just realized my current deployed version 0.1.2 is an old one and that I need to deploy a new one.

Maybe you can download all current code repo in the meantime.

**Comment by JulianSamarjiev** [comment]
I am able to update it to 0.2.0

**Comment by darul75** [comment]
You are true, npm has that current version but I forgot to create a release on github side as well. So then problem can come from the database not loaded correctly in memory ?

**Comment by JulianSamarjiev** [comment]
That could be. How would you recommend approaching this? Forking and putting it in the project manually?

**Comment by abhishek-b-agarwal-zz** [comment]
Hey @darul75 are you keeping the internal database updated?

**Comment by darul75** [comment]
Hello I guess latest update was made by someone in this pr https://github.com/darul75/satelize/commit/77350486127745692388423bfa4be9ac26bca592. It has been merged but I did not release any new version. You could probably update your own manually first.

**Comment by abhishek-b-agarwal-zz** [comment]
Is there any other API/NPM Library I can leverage to get user's location based on IP?
